src/internal/tester/restore_destination.go

```diff
@@ -1,11 +1,26 @@
 package tester
 
 import (
+	"strings"
+
 	"github.com/alcionai/corso/src/internal/common/dttm"
 	"github.com/alcionai/corso/src/pkg/control"
 )
 
-func DefaultTestRestoreDestination() control.RestoreDestination {
-	// Use microsecond granularity to help reduce collisions.
-	return control.DefaultRestoreDestination(dttm.SafeForTesting)
+const RestoreFolderPrefix = "Corso_Test"
+
+func DefaultTestRestoreDestination(namespace string) control.RestoreDestination {
+	var (
+		dest = control.DefaultRestoreDestination(dttm.SafeForTesting)
```

Comment:
Looks like we are overwriting dest on line 23 below. Should we remove this call?

Reply:
Nope, this will populate values beyond just the container name.